### ISSUE TITLE
Show resolved resource runs in run preview

### ISSUE THREAD

**Issue by gar1t** [guildai/guildai#64]
When an operation requires the output of another run, it's unclear what run is selected. In fact, the run is selected by Guild at runtime, after Guild shows the preview prompt. This leaves the user investigating the selected run *after* the downstream operation has started.

Desired behavior:

```
$ guild run train
You are about to run train
  prepared-data: abcd1234
```

**Comment by gar1t** [comment]
Fixed - pending 0.7 release.

**Comment by gar1t** [comment]
Fix is now available in 0.7.0.
